### PR TITLE
Fix sendSignedTransaction json rpc

### LINKED ISSUE #1044: [BUG] Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
how to reproduce:
```
yarn run test_integration_node
```

error logs:
```
...
      ain_sendSignedTransaction api
        ✓ accepts a transaction with unordered nonce (-1)
        ✓ accepts a transaction with numbered nonce
        ✓ accepts a transaction with account registration gas amount from nonce
        ✓ accepts a transaction without account registration gas amount from nonce
        ✓ accepts a transaction without account registration gas amount from balance
        ✓ accepts a transaction with account registration gas amount from balance (49ms)
        ✓ accepts a transaction without account registration gas amount from balance
        ✓ accepts a multi-set transaction with account registration gas amount from nonce
        ✓ accepts a multi-set transaction without account registration gas amount from nonce
        ✓ accepts a transaction with app creation gas amount
        ✓ rejects a transaction without app creation gas amount
        ✓ rejects a transaction that exceeds its size limit.
        ✓ rejects a transaction of missing properties.
        ✓ rejects a transaction in an invalid format.
        ✓ rejects a transaction with an invalid signature.
Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:372:5)
    at ServerResponse.setHeader (node:_http_outgoing:576:11)
    at ServerResponse.writeHead (node:_http_server:309:21)
    at /home/seo/ain-blockchain/node_modules/jayson/lib/server/middleware.js:51:15
    at Object.Utils.JSON.stringify (/home/seo/ain-blockchain/node_modules/jayson/lib/utils.js:290:3)
    at /home/seo/ain-blockchain/node_modules/jayson/lib/server/middleware.js:39:18
    at callback (/home/seo/ain-blockchain/node_modules/jayson/lib/server/index.js:241:22)
    at respond (/home/seo/ain-blockchain/node_modules/jayson/lib/server/index.js:292:9)
    at /home/seo/ain-blockchain/node_modules/jayson/lib/server/index.js:329:7
    at /home/seo/ain-blockchain/node_modules/jayson/promise/lib/method.js:43:19
      ain_sendSignedTransactionBatch api
        ✓ accepts a batch transaction
        ✓ rejects a batch transaction of invalid batch transaction format.
        ✓ accepts a batch transaction of under transaction list size limit. (117ms)
        ✓ rejects a batch transaction of over transaction list size limit.
        ✓ rejects batch transactions that cause over per-account transaction pool size limit. (354ms)
        ✓ rejects a batch transaction with a transaction that exceeds its size limit.
        ✓ rejects a batch transaction of missing transaction properties.
        ✓ rejects a batch transaction of invalid format.
        ✓ rejects a batch transaction with an invalid signature.
```

### PR DESCRIPTION
**Pull request by minsulee2**
It caused because of double done() is called.
Fix #1044

### CODE REVIEW

File: json_rpc/transaction.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ module.exports = function getTransactionApis(node, p2pServer) {
                 message: `Invalid transaction signature.`
               }
             }));
+            return;
           }
           const result = p2pServer.executeAndBroadcastTransaction(createdTx);
```
Comment (platfowner) [inline json_rpc/transaction.js:123]: thanks for fixing this!
what about adding this section into an 'else' block for readability instead?

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for fixing this!

**Comment by minsulee2** [comment]
Thanks! Merging now !

File: json_rpc/transaction.js (hunk 1 above)
Comment (minsulee2) [reply json_rpc/transaction.js]: Done!

**Pull request merged**